packages/react-server/src/ReactFlightServer.js

```diff
@@ -213,11 +213,8 @@ function patchConsole(consoleInst: typeof console, methodName: string) {
           1,
         );
         request.pendingChunks++;
-        // We don't currently use this id for anything but we emit it so that we can later
-        // refer to previous logs in debug info to associate them with a component.
```

Comment:
We don't do this. Instead, we associate the log with the component via the `owner` passed to the console chunk.